pages/index.tsx

```diff
@@ -1,26 +1,40 @@
 import { Button } from '@chakra-ui/react';
 import styled from '@emotion/styled';
-import Head from 'next/head';
 import Image from 'next/image';
-import { RiKakaoTalkFill } from 'react-icons/ri';
+
+import kakaoLoginImage from '../public/images/kakao_login_medium_wide.png';
 
 export default function Home() {
+  function loginWithKakao() {
+    window.Kakao.Auth.authorize({
+      redirectUri: 'http://localhost:3000/',
+    });
+  }
```

Comment:
`loginWithKakao`함수는 컴포넌트 밖으로 추출해도 괜찮을 것 같아요.

```suggestion

import kakaoLoginImage from '../public/images/kakao_login_medium_wide.png';

function loginWithKakao() {
    window.Kakao.Auth.authorize({
      redirectUri: 'http://localhost:3000/',
    });
  }

export default function Home() {
```

이렇게요 ㅎㅎ